### PR TITLE
Add domain

### PR DESCRIPTION
**Pull request by dntluchini**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by Stef-00012** [comment]
your domain has no records

**Review by dntluchini — COMMENTED** [review]
Added record and github page url

File: domains/dnt-luchini.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "dntluchini",
+        "email": "danteluchini@gmail.com"
+    },
+    "record": {
+        "URL": "https://dntluchini.github.io/Portfolio/",
```
Comment (DEV-DIBSTER) [inline domains/dnt-luchini.json:7]: Cannot have CNAME and URL together.

File: domains/dntluchini.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "owner": {
+        "username": "dntluchini",
+        "email": "danteluchini@gmail.com"
+    }
```
Comment (DEV-DIBSTER) [inline domains/dntluchini.json:5]: There's no records here.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Comment by dntluchini** [comment]
Any news?

**Pull request closed without merging**